antarest/core/interfaces/eventbus.py

```diff
@@ -14,6 +14,10 @@ class EventType:
     STUDY_VARIANT_GENERATION_COMMAND_RESULT = (
         "STUDY_VARIANT_GENERATION_COMMAND_RESULT"
     )
+    TASK_STARTED = "TASK_STARTED"
```

Comment:
actually this is a "task added", there is no guarantee (there is a threadpool with limited workers) that the task will run right after.